integration-tests/.relay-chain.mocharc.json

```diff
@@ -0,0 +1,16 @@
+{
+    "exit": true,
+    "extension": [
+        "ts"
+    ],
+    "parallel": false,
+    "require": [
+        "scaffolding/rootHooks.ts",
+        "scaffolding/extrinsicHelpers.ts"
+    ],
+    "loader": "ts-node/esm",
+    "spec": [
+        "./{,!(node_modules|load-tests)/**}/createMsa.test.ts"
+    ],
+    "timeout": 200000
```

Comment:
The timeout is increased so that `signAndSend()` has sufficient time to wait on the callback from the chain while the relay validates / seals each block

Reply:
Is that 3.3 minutes?  Is that too long?

Reply:
It may not be long enough for some tests in the replenishment suite to complete

Reply:
Interesting.  Why does validation/sealing take so long?  I thought blocks could be produced every 12 seconds or so?

Reply:
This likely will be able to be reduced for Frequency on Rococo. Testing with a local relay overshoots that target often. We can fine tune this

Reply:
Do you see a downside to a longer time out when testing with a Relay?

Reply:
I'm just trying to understand why there is slow performance.  The timeout needs to be set relative to the expected performance.

Reply:
I had to think about it for a minute, here is why: 
Some tests have multiple transaction per test. I am also wondering if the transaction may also not be guaranteed to make it into the current block.
```
  Create Accounts
    createMsa
      ✔ should successfully create an MSA account (47902ms)
```